other/js/websockify.js

```diff
@@ -26,9 +26,9 @@ var argv = require('optimist').argv,
 
 
 // Handle new WebSocket client
-new_client = function(client) {
+new_client = function(client, req) {
     var clientAddr = client._socket.remoteAddress, log;
-    console.log(client.upgradeReq.url);
+    console.log(req.url);
```

Comment:
Thanks. Changed accordingly in https://github.com/novnc/websockify/pull/281/commits/141ba0088ea80cca9d14fa9adaf5f9ca5a8b6d8a